solr/core/src/java/org/apache/solr/core/SolrResourceLoader.java

```diff
@@ -694,16 +694,32 @@ public <T> boolean addToCoreAware(T obj) {
     }
   }
 
-  void initCore(SolrCore core) {
```

Comment:
So there was `initCore(SolrCore)` and `inform(SolrCore)` - the former felt more like a typical setter to me, so I renamed the existing method, and named the two new methods in a similar fashion.